lib/vanagon/utilities.rb

```diff
@@ -308,5 +308,11 @@ def erb_file(erbfile, outfile = nil, remove_orig = false, opts = { :binding => b
       FileUtils.rm_rf erbfile if remove_orig
       outfile
     end
+
+    # Method to check for the sudo command if it exists
+    # Used to ensure builds inside containers that lack sudo binary don't fail
+    def sudo_bin
+      sudo_path = "$(command -v sudo)".strip
```

Comment:
Sorry, one more thing.  We can leave this function if you like (but remove assignment to a var and the `.strip` because they don't really do anything here), but I think we can also just put `$(command -v sudo)` directly where we're doing sudo across platforms (where you use `sudo_bin` above), since it's not very many places.